code/src/cljs/sixsq/nuvla/ui/notifications/spec.cljs

```diff
@@ -47,7 +47,9 @@
           :opt-un [::window
                    ::value
                    ::value-type]))
-(s/def ::metric utils-spec/nonblank-string)
```

Comment:
this line has been deleted by error?

Reply:
There is a definition on line 37. It was a duplication.

Reply:
👍